lighthouse.md

```diff
@@ -0,0 +1,66 @@
+# Project Lighthouse
+
+Project Lighthouse is conceptually linked to Project Carmel. They may appear as separate entities, but both projects can potentially share similar technical foundations. While Carmel is focused on debates, Lighthouse takes those principles and applies them to the management of a knowledge base. The underlying architecture can be similar, promoting synergies and allowing both projects to benefit from shared enhancements and updates.
+
+Draft spec, please provide feedback in the form of pull requests and issues to github.com/allinbits/carmel.
+
+## Initial terms
+
+ * All contributors agree on a curator or a panel of curators.
+ * All contributors commit to intellectual honesty, constructive debate, and the pursuit of truth.
+ * All contributors respect that this is an uncensored platform but are responsible for the veracity of their contributions.
+
+## Step 1. Determine the Discussion Points
+
+ * Any contributor can suggest a complex or controversial topic for discussion.
```

Comment:
In relation to the topics, which is your PoV of how Lighthouse deals with the topics? Similar to reddit in the sense of separating the topics but in the same world (not able to define if website or realm or any other product) so others can use other topics are resources or completely independent in different spheres so you can't mix them?

Reply:
Carmel is designed to frame trusted debates, while Lighthouse is intended to provide trusted, uncensored responses to intricate questions, similar to a credible Wikipedia.

With Carmel and Lighthouse's emphasis on open-source inspired, scientifically informed statements, we aim to create a rich network of linked statements with metadata, going beyond specific debates or topics.
Imagine git for statements.